models/brats_mri_axial_slices_generative_diffusion/scripts/download_brats_data.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) MONAI Consortium
```

Comment:
please remove this file. There are some errors of it (for example, `args.dataset_dir` is "@bundle_root" rather than ".".
Instead of fixing the issue of it, I suggest to not provide the download script. Like other bundles such as `spleen_ct_segmentation`, I think provide the reference link is enough for users to prepare the dataset.
I made the change in the 3d bundle PR, feel free to refer it.

Reply:
Yes, Thank you!